pkg-manager/link-bins/src/index.ts

```diff
@@ -72,15 +73,16 @@ export async function linkBinsOfPkgsByAliases (
     (await Promise.all(
       depsAliases
         .map((alias) => ({
+          pkgAlias: alias,
           depDir: path.resolve(opts.modulesDir, alias),
           isDirectDependency: directDependencies?.has(alias),
           nodeExecPath: opts.nodeExecPathByAlias?.[alias],
         }))
         .filter(({ depDir }) => !isSubdir(depDir, binsDir)) // Don't link own bins
-        .map(async ({ depDir, isDirectDependency, nodeExecPath }) => {
+        .map(async ({ pkgAlias, depDir, isDirectDependency, nodeExecPath }) => {
           const target = normalizePath(depDir)
           const cmds = await getPackageBins(pkgBinOpts, target, nodeExecPath)
-          return cmds.map((cmd) => ({ ...cmd, isDirectDependency }))
+          return cmds.map((cmd) => ({ ...cmd, isDirectDependency, ownAlias: cmd.name === pkgAlias, pkgAlias }))
```

Comment:
I am not sure we need to consider aliases.

Reply:
I have removed alias from consideration